container-images/tcib/base/tobiko/run_tobiko.sh

```diff
@@ -30,8 +29,9 @@ TOBIKO_LOGS_DIR_NAME=${TOBIKO_LOGS_DIR_NAME:-"tobiko"}
 [ ! -z ${TOBIKO_NUM_PROCESSES} ] && export TOX_NUM_PROCESSES=${TOBIKO_NUM_PROCESSES}
 
 pushd ${TOBIKO_DIR}
-git clone https://opendev.org/x/tobiko
+sudo chown tobiko:tobiko -R tobiko
 pushd tobiko
+[ ! -z ${TOBIKO_UPDATE_REPO} ] && git pull --rebase
```

Comment:
We probably need an option for this in the Tobiko CR, right?

Reply:
yes, but it will be disabled by default, so no rush